django/db/models/sql/query.py

```diff
@@ -988,6 +988,16 @@ def prepare_lookup_value(self, value, lookups, can_reuse, allow_joins=True):
             pre_joins = self.alias_refcount.copy()
             value = value.resolve_expression(self, reuse=can_reuse, allow_joins=allow_joins)
             used_joins = [k for k, v in self.alias_refcount.items() if v > pre_joins.get(k, 0)]
+        elif isinstance(value, (list, tuple)):
```

Comment:
Is this code self-explanatory enough not to need any comments? I have no clue...